cabal.project

```diff
@@ -35,7 +35,7 @@ package *
 
 -- Warnings as errors for local packages
 program-options
-  ghc-options: -Werror
+  ghc-options: -Wwarn
```

Comment:
Must: not change the project config. See https://github.com/input-output-hk/hydra/blob/master/CONTRIBUTING.md#building--testing for how to have a project.local which does not fail on warnings.